apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsTransactionNameInstrumentation.java

```diff
@@ -79,6 +81,7 @@ private static void setTransactionName(@SimpleMethodSignature String signature,
                     }
                 }
                 transaction.withName(transactionName, PRIO_HIGH_LEVEL_FRAMEWORK, false);
+                transaction.getTraceContext().setFrameworkName(FRAMEWORK_NAME);
```

Comment:
Could we use `co.elastic.apm.agent.util.VersionUtils#getVersionFromPomProperties` to get the version? Remember to cache the version as this method is quite expensive. Probably best to cache with a `WeakConcurrentMap` with a class such as `javax.ws.rs.GET.class` as the key. That's necessary as there could be multiple JAX-RS versions in different `war`s in an app server.

Reply:
Great! It's working!

Reply:
added. I added cache into JaxRsTransactionNameInstrumentation.